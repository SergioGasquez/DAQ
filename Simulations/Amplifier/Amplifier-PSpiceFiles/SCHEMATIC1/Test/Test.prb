[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS DC_SWEEP
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN CURSORS
CURSORNUMDIGITS 5
BEGIN CURSOR 1
CURSOR_X 2.48999999999999
CURSOR_Y 13.1752729006635
ANAPLOT 1
ANATRACE 0
END CURSOR 1
BEGIN CURSOR 2
CURSOR_X -2.5
CURSOR_Y -14.2589295435192
ANAPLOT 1
ANATRACE 0
END CURSOR 2
END CURSORS
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
BEGIN TRACE V(Vo)
MARKERID 3
END TRACE V(Vo)
BEGIN TRACE V(V1:+)
MARKERID 4
END TRACE V(V1:+)
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST SESSION
