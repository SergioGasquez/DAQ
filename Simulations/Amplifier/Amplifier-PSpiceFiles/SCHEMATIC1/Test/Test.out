
**** 04/05/18 19:19:50 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\Amplifier\Amplifier-PSpiceFiles\SCHEMATIC1\Test.sim


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "Test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\sergi\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.DC LIN V_V1 -2.5 2.5 0.01 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source AMPLIFIER
X_U2B         N01850 N01868 N01772 N01912 VO LM358
R_R4         N01868 VO  50k TC=0,0 
V_V3         N01912 GND -15Vdc
V_V1         N01850 GND 5Vdc
V_V4         N01772 GND 15Vdc
R_R5         GND N01868  10k TC=0,0 

**** RESUMING Test.cir ****
.END

**** 04/05/18 19:19:50 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\Amplifier\Amplifier-PSpiceFiles\SCHEMATIC1\Test.sim


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U2B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U2B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


**** 04/05/18 19:19:50 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\Amplifier\Amplifier-PSpiceFiles\SCHEMATIC1\Test.sim


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U2B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


**** 04/05/18 19:19:50 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\Amplifier\Amplifier-PSpiceFiles\SCHEMATIC1\Test.sim


 ****     Current Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_U2B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U2B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 



          JOB CONCLUDED

**** 04/05/18 19:19:50 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\Amplifier\Amplifier-PSpiceFiles\SCHEMATIC1\Test.sim


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .05
